frontend/src/components/Spotiverse/Spotiverse.tsx

```diff
@@ -5,72 +5,81 @@ import { getUsersLikedSongs, getTopGlobalSongs } from "../../spotifyDataAccess";
+import { TestData } from "./test-data";
+import { Axis } from "../Axes/Axes";
+import "./Spotiverse.css";
 
 export interface ISpotiverseProps {
   spotifyUser?: ISpotifyUser;
 }
 
-interface Axes {
-  //these should be a type of all the options we have for graphing
-  // i.e. valence, loudness, tempo, etc...
-  X: any;
-  Y: any;
-  Z: any;
-
-}
-
-
-export const Spotiverse: React.FC<ISpotiverseProps> = (props) => { 
-  const [songs, setSongs] = useState<Song[]>(getTopGlobalSongs);
-  const appState = useContext(AppStateContext);
+export const Spotiverse: React.FC<ISpotiverseProps> = (props) => {
+  const [songs, setSongs] = useState<Song[]>([]);
   const [selectedSong, setSelectedSong] = useState<Song>(null);
   const [isLoading, setLoading] = useState(true);
 
-  const [axisChange, setAxisChange] = useState(null) //this will be a boolean flip that we will listen too in three engine.tsx, every time it changes we should update particles locations
+  const [axisChange, setAxisChange] = useState(null); //this will be a boolean flip that we will listen too in three engine.tsx, every time it changes we should update particles locations
 
   // on mount (and when token changes), kickoff a request for the users liked songs
   // update state when they come down
   useEffect(() => {
-    if (props.spotifyUser?.token)
-      getLikedSongs()
-  }, [props.spotifyUser?.token])
+    if (props.spotifyUser?.token) getLikedSongs();
+  }, [props.spotifyUser?.token]);
 
   const getLikedSongs = async () => {
     //this could be run as a web worker when we start up the app
     const songs = await getUsersLikedSongs(props?.spotifyUser?.token);
     setSongs(songs);
-    setLoading(false)
-  }
+    setLoading(false);
+  };
 
   const setSong = (song: Song) => {
-    setSelectedSong(song)
-  }
-
+    setSelectedSong(song);
+  };
 
   const renderAxis = () => {
     return (
       <div className="AxesWrapper">
-        <Axis axisChange={axisChange} color="#0b933b" setAxisChange={setAxisChange} axis="X" defaultValue="speechiness" />
-        <Axis axisChange={axisChange} color="#229d9d" setAxisChange={setAxisChange} axis="Y" defaultValue="acousticness" />
-        <Axis axisChange={axisChange} color="#c7e7a1" setAxisChange={setAxisChange} axis="Z" defaultValue="valence" />
+        <Axis
+          axisChange={axisChange}
+          color="#0b933b"
+          setAxisChange={setAxisChange}
+          axis="X"
+          defaultValue="speechiness"
+        />
+        <Axis
+          axisChange={axisChange}
+          color="#229d9d"
+          setAxisChange={setAxisChange}
+          axis="Y"
+          defaultValue="acousticness"
+        />
+        <Axis
+          axisChange={axisChange}
+          color="#c7e7a1"
+          setAxisChange={setAxisChange}
+          axis="Z"
+          defaultValue="valence"
+        />
       </div>
-    )
-  }
+    );
+  };
 
   return (
     <>
-        <div style={{ display: "flex", flexDirection: "row", height: "92vh" }}>
```

Comment:
ok i think i should have kept this right @mtrent6

Reply:
from what I remember yeah, that should set the height of the graph and songlist and make it not overflow over the header